docs/getting-started/endgame-sensor-FDA-ven.asciidoc

```diff
@@ -0,0 +1,78 @@
+
+Here are the following Full Disk Access requirements for specific versions of macOS:
+
+
+- `13.0` and above: You cannot install the sensor without allowing it to load a <<system-extension, system extension>>. During installation, you'll be prompted to go to System Preferences and approve it. Upon approval, a second prompt appears to enable Network Filtering. Approve this final prompt to proceed.
++
+You must also grant Full Disk Access to `com.endgame.systemextension`.
+
+- `13` and above: Grant the <<endpoint-endgame-sensor, esensor file>> Full Disk Access.
+
+NOTE: The following instructions apply to the Endgame sensor only. To see requirements for the {elastic-endpoint}, refer to <<deploy-elastic-endpoint>>.
+
+[discrete]
+[[system-extension-ven]]
+== Approve the system extension for the Endgame sensor
+
+To fully protect endpoints from malware and other cybersecurity threats when using Elastic Endgame with system extensions, you must enable the system extension during installation on macOS Big Sur (11.0) and later.
+
+When you receive the prompt to approve loading the system extension:
+
+. Open the *System Settings* application.
+. Select *Privacy & Security*.
+. In the right pane, scroll down to the Security section. Click **Allow** to allow the Endgame system extension to load.
++
+image::fda/endgame_allow_sys_ext_ven.png[]
++
+. Enter your username and password and click **Modify Settings** to save your changes.
+
+[discrete]
+[[allow-network-filter-content-ven]]
+== Approve network content filtering for Endgame
+
+After successfully loading the Endgame system extension, an additional message appears, asking to allow Endgame to filter network content.
+
+[role="screenshot"]
+image::fda/endgame_allow_network_filter_ven.png[]
+
+Click *Allow* to enable content filtering for the Endgame system extension. Without this approval, Endgame cannot receive network events and, therefore, cannot enable network-related features such as <<host-isolation-ov, host isolation>>.
+
+[discrete]
+[[endpoint-endgame-sensor-ven]]
+== Enable Full Disk Access for the Endgame sensor
+
+For the Endgame sensor to detect events from a macOS host, you must enable Full Disk Access for the `esensor` file, which appears once you've downloaded the sensor on your host.
+
+. Open the *System Settings* application.
+. Select *Privacy & Security*.
+. From the right pane, select *Full Disk Access*.
++
+[role="screenshot"]
+image::install-endpoint-ven/select_fda_ven.png[Select Full Disk Access]
++
+. Click the *+* button to view *Finder*.
+. The system may prompt you to enter your username and password if you haven't already.
++
+[role="screenshot"]
+image::fda/endgame_enter_login_details_to_confirm_ven.png[]
++
+. Navigate to `/Library/Endgame`, then select the `esensor` file.
+. Click *Open*.
+. In the *Privacy* tab, confirm that the `elastic-agent` or `esensor` file appears in the list of applications with Full Disk Access permissions.
+
```

Comment:
Add screenshot with the final result 
![Screenshot 2023-08-02 at 13 08 23](https://github.com/elastic/security-docs/assets/81707160/b3a327c5-668b-4616-ab4f-5dcf5906f7b4)

Reply:
Thanks for this, @ricardoungureanu! I added it in [871892f](https://github.com/elastic/security-docs/pull/3565/commits/871892f19f42e3b567692c9690b49ae0bd8eab51).